charts/bungee/templates/configmap.yaml

```diff
@@ -0,0 +1,14 @@
+{{- if .Values.bungeeCord.config }}
+apiVersion: v1
+kind: ConfigMap
+metadata:
+  name: {{ include "bungee.fullname" . }}-config
+  labels:
+    app: {{ template "bungee.fullname" . }}
+    chart: "{{ .Chart.Name }}-{{ .Chart.Version }}"
+    release: "{{ .Release.Name }}"
+    heritage: "{{ .Release.Service }}"
+data:
+  config.yml: |
+{{ toYaml .Values.bungeeCord.config | indent 4 }}
+{{- end }}
```

Comment:
> probably should remove the toYaml from the configmap to open it up to other file formats (like velocity's .toml)

Yeah, I wonder how well it would work to have a more generalized "config files" section in the yaml, which would be key-value mappings of filenames to content to be indented into the configmap.

Reply:
Agree, once it's merged I'll raise an issue to work on that. The current implementation works perfectly for Waterfall/BungeeCord which covers the vast majority of the use cases.